### ISSUE TITLE
Providing folder structure in providedFiles

### ISSUE THREAD

**Issue by kitze** [Noviny/react-codesandboxer#11]
Hello! 

First of all, great package, thank you for your work. 

I'm trying to use it for my workshops but I'm missing one thing, uploading folder structure instead of only files with `key` and `content` values. Is that possible with the current API or would it need some changes?

Thanks

**Comment by Noviny** [comment]
It would need changes, unfortunately.

Note that

```
files={{
  'some/path/to/file.js': {
     content: '/* Some file contents */'
  }
}}
```

works.

Passing in files, we uploading directly that object to codesandboxer, so want to keep it in that format. Do you think a utility to flatten a folder structure would make this work for you? Something like?

```
converterFunction({
  some: {
    place: {
      'file.js': '/* string of file contents */'
    }
  }
})
```

to output:

```
{
  'some/place/file.js': {
    content: '/* string of file contents */'
  }
}
```

**Comment by kitze** [comment]
@Noviny I didn't know that was possible. I built my own converter function so I got the folder structure now 🎊️ 

The only problem is my examples already have an "index.js" file that's located in the "src" folder (they're CRA apps), and I don't want the example.js file there. Is there a way to remove the example.js file, the /index.js file and to point at src/index.js as the main file?

Thanks so much for your help.
